web/src/main/resources/spring/web-common-ctx.xml

```diff
@@ -56,5 +56,9 @@
     <bean id="invalidRequestProcessor" class="edu.softserve.zoo.validation.impl.InvalidRequestProcessorImpl"/>
 
     <context:component-scan
-            base-package="edu.softserve.zoo.controller.rest, edu.softserve.zoo.converter, edu.softserve.zoo.advice"/>
+            base-package="edu.softserve.zoo.controller.rest,
```

Comment:
assuming that is temporary and you wouldn't use such an approach to pull a module lying below